.github/ISSUE_TEMPLATE/yank-buildpack.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Yank Buildpack
+about: Propose the removal of a buildpack from the registry index
+title: ADD {BUILDPACK_ID}@{VERSION}
```

Comment:
Should be `title YANK  {BUILDPAK_ID}@{VERSION`

<img width="425" alt="Screen Shot 2020-10-28 at 4 32 15 PM" src="https://user-images.githubusercontent.com/1181106/97503666-15890c00-193b-11eb-8d75-73b0ccc56121.png">